cl/lib/timezone_helpers.py

```diff
@@ -31,7 +32,14 @@ def localize_date_and_time(
     :return: A tuple of date_filed and time_filed or None if no time available.
     """
     if isinstance(date_filed, datetime):
-        datetime_filed_local = convert_to_court_timezone(court_id, date_filed)
+        if is_naive(date_filed):
+            datetime_filed_local = localize_naive_datetime_to_court_timezone(
+                court_id, date_filed
+            )
+        else:
+            datetime_filed_local = convert_to_court_timezone(
+                court_id, date_filed
+            )
```

Comment:
I think this is probably not sufficiently clear to the reader, and the function docstring should be updated.